examples/goldilocks_models/initial_guess.cc

```diff
@@ -0,0 +1,195 @@
+//
+// Created by jianshu on 3/25/20.
+//
+#include "examples/goldilocks_models/initial_guess.h"
+
+namespace dairlib::goldilocks_models {
+//edited by Jianshu to try a new way of setting initial guess
+
+MatrixXd get_theta_scale(const string directory, int iter){
+//    VectorXd initial_theta = readCSV(directory + to_string(0) + string("_theta_s.csv"));
+    VectorXd current_theta = readCSV(directory + to_string(iter) + string("_theta_s.csv"));
+    VectorXd theta_scale = 1*VectorXd::Ones(current_theta.rows());
+    MatrixXd diag_theta_scale = theta_scale.asDiagonal();
+    return diag_theta_scale;
+}
+
+MatrixXd get_gamma_scale(int gamma_length, double min_sl, double max_sl, double min_gi, double max_gi,
+        double min_tr, double max_tr){
+    VectorXd gamma_scale(gamma_length);
+    gamma_scale << 1, 1, 2;
```

Comment:
Why not use `min_sl`, `max_sl`, `max_gi`, ... to get the scaling?

Reply:
I moved the scale created by `min_sl, max_sl, max_gi, ...` into the function set_initial_guess.